src/Core/src/Animations/AnimationManager.cs

```diff
@@ -1,106 +1,114 @@
 ﻿using System;
 using System.Collections.Generic;
 
-namespace Microsoft.Maui.Animations
+namespace Microsoft.Maui.Animations;
+
+/// <inheritdoc/>
```

Comment:
what does this do? also what's with the namespace change do we need it? .. it will mess with history if change all files like this

Reply:
It uses the file-scoped namespaces so we lose one level of indentation. History will definitely be a mess, but it will be a mess anyway with all the API documentation changes that I will be doing 😅

Reply:
Just touching docs and not any other changes right now!